src/domUpdates.js

```diff
@@ -46,6 +47,8 @@ const domUpdates = {
       3: $('#player-three-input').val(),
     };
     const prompt = {
+      93: `Now enter your final answers!`,
```

Comment:
I like this object that represents the prompts. Just wondering, why the use of these specific numbers?

Reply:
@robbiejaeger when first I wrote this object I started with 100, just wanted to start with three digits. Then I realized I had prompts need to go before the first one.. and then it became messy like that :)